src/Test/Configuration.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Zenstruck\Foundry\Test;
+
+use Faker;
+use Psr\Container\ContainerInterface;
+use Psr\Container\NotFoundExceptionInterface;
+use Zenstruck\Foundry\Factory;
+use Zenstruck\Foundry\Manager;
+use Zenstruck\Foundry\StoryManager;
+
+/**
+ * @author Kevin Bond <kevinbond@gmail.com>
+ */
+final class Configuration
+{
+    /** @var callable|null */
+    private static $instantiator;
+    private static ?Faker\Generator $faker = null;
+
+    public static function setInstantiator(callable $instantiator): void
+    {
+        self::$instantiator = $instantiator;
+    }
+
+    public static function setFaker(Faker\Generator $faker): void
+    {
+        self::$faker = $faker;
+    }
+
+    public static function bootFactory(ContainerInterface $container): Manager
+    {
+        try {
+            $manager = $container->get(Manager::class);
+        } catch (NotFoundExceptionInterface $e) {
+            // bundle not enabled
+            Factory::boot($manager = new Manager($container->get('doctrine'), new StoryManager([])));
```

Comment:
Possible wtf here if you intend to use the bundle but it’s not enabled or misconfigured. I’m not sure if that’s a real problem... as most users won’t have much config on the Manager service anyways. But, you could force non-bundle users to opt into on-bundle usage with some Configuration::withoutBundle(). But I’m not sure this is needed

Reply:
Yeah, good call - I'll re-evaluate.